README.md

```diff
@@ -40,13 +40,15 @@ sure that we're all on the same page.
   done the work _on theirs directly_. [Here][pr] is a great example of a pull
   request on the `Ruby` codebase.
 
-Let's go over a conceptual, hypothetical example. It's OK if this feels a bit confusing at
-first. You'll work through this countless times and eventually your brain and
-fingers will both grasp what's going on. Let's look at the following example:
+We're going to go go over a conceptual, hypothetical example. THIS IS ONLY AN 
+EXAMPLE. It's OK if this feels a bit confusing at first. You'll work through 
```

Comment:
the point is to avoid them actually cloning from `learn-co-students/awesome-repo`. I think this is shouting at them but not modifying their behavior.

"In the following examples we'll use (url). We're not expecting you to type along and execute the commands (i.e. don't `git clone url`, we want you to read the 

etc.